web-app/src/app/store/profile-reducer.ts

```diff
@@ -93,6 +93,7 @@ export const userProfileSlice = createSlice({
       state.status = 'unauthenticated';
       state.isSignedInWithProvider = false;
       state.isAppRefreshing = false;
+      state.user = undefined;
```

Comment:
This is not related to the fix. However, I noticed the user information stayed in the store after logout. This might lead to unexpected behavior after completing the logout and before the anonymous login succeeds.